node/pkg/node/node.go

```diff
@@ -33,6 +33,11 @@ const (
 	// One observation takes roughly 0.1ms to process on one core, so the whole queue could be processed in 1s
 	inboundObservationBufferSize = 10000
 
+	// inboundBatchObservationBufferSize configures the size of the batchObsvC channel that contains batches of observations from other Guardians.
+	// Since a batch contains many observations, the guardians should not be publishing too many of these, so we can keep the channel small. With
+	// 19 guardians, we would expect 19 messages per second during normal operations. This gives us plenty of extra room.
```

Comment:
Does this math change with the new batching exception? I would expect this channel may need to be a bit larger to accomodate for several message in a second being included in a batch of their own.

Reply:
Increased it to 1000 and updated the comment.